lib/mix/tasks/nerves.release.init.ex

```diff
@@ -14,7 +17,11 @@ defmodule Mix.Tasks.Nerves.Release.Init do
 
   @impl true
   def run(args) do
-    template_path = Path.join(["#{:code.priv_dir(:nerves)}", "templates", "release.eex"])
-    Mix.Task.run("release.init", args ++ ["--template", template_path])
+    if use_distillery?() do
+      template_path = Path.join(["#{:code.priv_dir(:nerves)}", "templates", "release.eex"])
+      Mix.Task.run("release.init", args ++ ["--template", template_path])
```

Comment:
@mobileoverlord Distillery 2.1.0 introduced a breaking change, the task name changed to `distillery.init`. See: https://github.com/bitwalker/distillery/issues/694

Reply:
Updated